ArticleTemplates/assets/js/bootstraps/common.js

```diff
@@ -417,6 +423,15 @@ define([
         document.body.dataset.fontSize = replacementInt[2];
     }
 
+    function lineHeightResize(current, replacement) {
```

Comment:
Can you give examples of `current` and `replacement` so I can see what class names are being swapped in/out?

Reply:
Current could be: `line-height-0`
Replacement could be: `line-height-1`